TVC_MVP/CountyPresenter.swift

```diff
@@ -92,18 +91,15 @@ class CountyPresenter: CountyPresenterProtocol {
     }
 
     private func generateAlertMessage(county: County) -> String {
-        var newbornString: String = ""
-        var populationString: String = ""
-        let id = Constants.alertTitleBlock + String(county.countyID)
-        let extras = county.countyExtras
-        if extras != nil {
+        var id = Constants.alertTitleBlock + String(county.countyID)
+        if county.hasExtras() {
```

Comment:
You are still doing the same, just added an extra method. That's not what I meant.